controllers/comments.controllers.js

```diff
@@ -0,0 +1,16 @@
+const { fetchArticlesId } = require("../models/articles.models");
+const { selectCommentsByArticleId } = require("../models/comments.models");
+
+
+exports.getCommentsByArticleId = (req, res, next) => {
+    const {article_id} = req.params
+    Promise.all([
```

Comment:
this code is less likely to introduce bugs if you use a sequential pattern here: first check if the article exists, then grab the comments.